### PR TITLE
[WIP] Adding Ink plugin

### PR DESCRIPTION
**Pull request by maestro321**

### CODE REVIEW

**Comment by maestro321** [comment]
@edisinovcic Ink has been added to plugins.

**Comment by edisinovcic** [comment]
@maestro321 tnx, @LianaHus can someone of you guys approve this.

Tnx :)

File: src/remixAppManager.js [hunk 1]
```diff
@@ -202,6 +202,18 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://zokrates.blockchain-it.hr/zokrates.svg',
       location: 'sidePanel'
     }
+    const ink = {
+      name: 'Ink',
+      displayName: 'Ink',
+      description: 'ink! is smart contracts tool using the Rust programming language.',
+      documentation: 'https://staging.ink-remix.blockchain-it.hr/',
```
Comment (LianaHus) [inline src/remixAppManager.js]: please update the link to docs

File: src/remixAppManager.js [hunk 2]
```diff
@@ -202,6 +202,18 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://zokrates.blockchain-it.hr/zokrates.svg',
       location: 'sidePanel'
     }
+    const ink = {
```
Comment (yann300) [inline src/remixAppManager.js]: Could you add the plugin reference directly in https://github.com/ethereum/remix-plugins-directory ?
cc @GrandSchtroumpf

Comment (edisinovcic) [reply src/remixAppManager.js]: Sure.

In here: https://github.com/ethereum/remix-plugins-directory/tree/master/plugins
and here: https://github.com/ethereum/remix-plugins-directory/blob/master/build/profile.json

Anywhere else?

Comment (yann300) [reply src/remixAppManager.js]: just add a new folder in there https://github.com/ethereum/remix-plugins-directory/tree/master/plugins
with the profile.json

Comment (edisinovcic) [reply src/remixAppManager.js]: Will do, tnx!

**Pull request closed without merging**